packages/common/src/index.js

```diff
@@ -1,4 +1,7 @@
+import * as _Utils from './utils'
+
 export { default as Modes } from './modes'
 export { default as NuxtConfig } from './nuxt.config'
 export { default as Options } from './options'
 export * from './utils'
+export const Utils = _Utils
```

Comment:
Now we export an object `Utils` includes all functions in `./utils` and also all functions on `index`, it looks kind of confusing, we'd better keep only one behaviour.

Reply:
@clarkdo gotcha, makes sense. I'll modify.